frontend/packages/text-editor/src/TextEditor.module.css

```diff
@@ -53,6 +53,19 @@
   justify-content: space-between;
 }
 
+.TextEditor__topRow__filter {
```

Comment:
Should we maybe follow the same way of naming css classes as we do in the rest of the code, using camelCase?😄